### PR TITLE
[DEV-1] Add support of multiple team keys

### PR DESCRIPTION
**Pull request by taleh007**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as core from '@actions/core'
 
-import Runner from './runner'
+import { default as Runner, Inputs } from './runner'
```
Comment (HolyWalley) [inline src/main.ts:3]: you know you can do `import Runner, { Inputs } from './runner'`

File: src/utils.ts [hunk 2]
```diff
@@ -16,23 +20,27 @@ const INPUT_KEYS = {
...
+  const issueIdentifiers = core.getInput(INPUT_KEYS.ISSUE_IDENTIFIERS).split(',').filter(Boolean)
+
+  return teamKeys.map(teamKey => {
+    const issueNumbers = issueIdentifiers
+      .filter(identifier => identifier.startsWith(`${teamKey}-`))
+      .map(identifier => {
+        return parseInt(identifier.replace(`${teamKey}-`, ''), 10)
+      })
+
+    return { apiKey, teamKey, transitionTo, issueNumbers, addLabels, removeLabels, transitionFrom, filterLabel }
```
Comment (HolyWalley) [inline src/utils.ts:44]: you don't need this return now, right?

File: src/utils.ts [hunk 3]
```diff
@@ -1,6 +1,10 @@
 import * as core from '@actions/core'
 import { Inputs } from './runner'
 
+export interface InputsGroup {
+  [key: string]: Inputs
+}
+
```
Comment (HolyWalley) [inline src/utils.ts:7]: not used, right?

File: src/utils.ts [hunk 4]
```diff
@@ -16,23 +20,27 @@ const INPUT_KEYS = {
...
-  return { apiKey, teamKey, transitionTo, issueNumbers, addLabels, removeLabels, transitionFrom, filterLabel }
+  const filterLabel = core.getInput(INPUT_KEYS.FILTER_LABEL)
+  const issueIdentifiers = core.getInput(INPUT_KEYS.ISSUE_IDENTIFIERS).split(',').filter(Boolean)
+
+  return teamKeys.map(teamKey => {
+    const issueNumbers = issueIdentifiers
+      .filter(identifier => identifier.startsWith(`${teamKey}-`))
+      .map(identifier => {
+        return parseInt(identifier.replace(`${teamKey}-`, ''), 10)
+      })
```
Comment (HolyWalley) [inline src/utils.ts:42]: 👍

**Review by HolyWalley — APPROVED** [review]

File: src/main.ts [hunk 5]
```diff
@@ -9,9 +9,16 @@ import { parseInputs } from './utils'
...
 export async function run(): Promise<void> {
   try {
-    const inputs = parseInputs()
-    const runner = new Runner(inputs.apiKey)
-    await runner.run(inputs)
+    const inputs: Inputs[] = parseInputs()
+
+    const runners: Promise<void>[] = inputs.map(async input => {
+      const runner = new Runner(input.apiKey)
+      return runner.run(input)
```
Comment (yard) [inline src/main.ts:16]: What will happen if an input has an empty list of issues (e.g. you configure it with 2 teams, BE and DEVX, but only BE-XXX issue identifiers are passed in)?

**Review by HolyWalley — APPROVED** [review]

**Pull request merged**